packages/client/package.json

```diff
@@ -14,12 +14,12 @@
   "homepage": "https://github.com/web3-storage/web3.storage#readme",
   "scripts": {
     "test": "npm-run-all -p -r mock:api test:all",
-    "test:all": "run-s test:web test:cjs test:esm test:size",
-    "test:size": "bundlesize",
+    "test:all": "run-s test:web test:cjs test:esm",
+    "test:size": "npx bundlesize",
```

Comment:
`npx` so that `npm install` in ndoe.js 20 doesn't fail because native module can't be compiled.